packages/namekit-react/src/components/Link.tsx

```diff
@@ -8,11 +8,15 @@ export interface LinkProps extends React.ComponentPropsWithoutRef<"a"> {
   asChild?: React.ReactElement;
 }
 
+interface LinkComponent extends React.FC<LinkProps> {
+  ExternalIcon: typeof ExternalIcon;
+}
+
 const linkBaseClasses = "nk-transition";
 
 const variantClasses = {
   primary:
-    "nk-text-black nk-underline sm:nk-underline-offset-[4px] sm:nk-transition-all sm:nk-duration-200 sm:hover:nk-underline-offset-[2px]",
+    "nk-text-current nk-underline sm:nk-underline-offset-[4px] sm:nk-transition-all sm:nk-duration-200 sm:hover:nk-underline-offset-[2px]",
```

Comment:
Could we add a story to the storybook showing an example where the background is dark and the text is light?

Reply:
The custom class name is the example for this 😅

Reply:
I added an explicit example though.